Velocity_Model/scripts/get_z_values.sl

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# script version: slurm
+#
+# Run on Maui
```

Comment:
Again, I don't think this is the right repo for slurm scripts.

Aside, get_z wasn't really multiprocessed, let alone MPI, so it seems weird to run on multiple nodes of Maui. Would mahuika be more suited for this kind of job?

Reply:
This takes lots of memory (as explained in readme.md), tried Mahuika's standard/large memory nodes to no avail. This configuration on Maui was the only successful one out of a number of options I tested.

Reply:
If it's running on multiple nodes, can it use the memory from both nodes without MPI?

Reply:
Yes, I considered the use of hypocentre too, but chose not too due to the inconvenience invoiced with moving data around.  Besides, you don't seem to realise that I don't use MPI. This is SLURM job arrays.